### DISCUSSION TITLE
How I can change IC2 config uu-values?

### DISCUSSION THREAD

**Discussion by Elinand** [EnigmaticaModpacks/Enigmatica2Expert#2132] [Q&A] [answered]
Predefined values are not changed and new ones are not added
![Screenshot_4](https://github.com/EnigmaticaModpacks/Enigmatica2Expert/assets/41586602/437adce3-3e56-4fe4-aafb-034a67168ec7)

**Comment by NielsPilgaard** [comment]
How are you checking whether values are added/changed? :)

**Reply by Elinand** [reply]
In tab "mod option" for adding and change, then use JEI to make sure

**Comment by NielsPilgaard** [comment]
We added the recipes in JEI, so updating the uu values in the ic2 config wouldn't actually change them in JEI :)

**Reply by Elinand** [reply]
Ok, how did you do that?

**Reply by NielsPilgaard** [reply, marked as answer]
Like this: https://github.com/EnigmaticaModpacks/Enigmatica2Expert/blob/master/scripts/Tooltips.zs#L522
